docs/source/known-issues.rst

```diff
@@ -23,6 +23,16 @@ HIP
 
   gcc 7, 9, and later do not have this issue.
 
+SYCL
+====
+
+- Multiple of the Kokkos algorithm functions use third-party libraries like oneDPL.
+  When using these, Kokkos doesn't control the kernel launch and thus the user has to make sure that all arguments
+  that are forwarded to the TPL satisfy the sycl::is_device_copyable trait to avoid compiler errors. This holds true in particular
+  for comparators used with Kokkos::sort in Kokkos versions prior to 4.3. The best advice to give is to make sure the respective
+  parameters are trivially-copyable. If this isn't possible, sycl::is_device_copyable should be specialized and users should make
+  sure to use raw pointers instead of Kokkos::Views.
```

Comment:
Can you add a code example and include the compile error so that it can be easily found by users?

Reply:
```C++
Kokkos::sort(exec, values, KOKKOS_LAMBDA(int i, int j) { return keys(i) < keys(j); });  // error blah with SYCL

Reply:
25284b5d

Reply:
25284b5d